src/select/DataSelect.tsx

```diff
@@ -101,7 +101,7 @@ export const DataSelect = <TNode extends object, TNodeKey extends keyof TNode>({
       return (
         <ListBox>
           <ListBoxItem key="loading" size={size}>
-            <Spinner />
+            <Spinner borderColor="primary.foreground" borderTopColor="white" />
```

Comment:
Why is the color set here if we have the default?

Reply:
We've white borderColor... we're using spinner elsewhere as well

Reply:
See what the most sensible default should be (most used) and make that the default

Reply:
I've done that.